client/Encounter/Encounter.ts

```diff
@@ -208,18 +208,6 @@ export class Encounter {
 
         const nextCombatant = this.Combatants()[nextIndex];
 
-        this.durationTags
```

Comment:
While I agree we can stop removing these at runtime, I think it would make sense to add logic to SaveEncounter to strip out tags with negative durations.

Reply:
Good point. Latest commits address this